CHANGELOG.md

```diff
@@ -2,20 +2,21 @@
 
 ## Unreleased
 
-- [[#42]](https://github.com/IronCoreLabs/gridiron/pull/42)
-  - Update to criterion 0.4 (Test only change)
-  - Change MSRV to Rust 1.57.0
+- [[#44]](https://github.com/IronCoreLabs/gridiron/pull/44) - Change MSRV to Rust 1.60.0
```

Comment:
We might not want to merge these onto one line - there might be things that are processing the changelogs looking for the PR lines?

Reply:
Agree that we should be consistent here.

Reply:
I just looked back and this is how we've done single line changelog entries in the past.